rai-sdk/src/main/java/com/relationalai/Client.java

```diff
@@ -734,13 +736,16 @@ public TransactionAsyncResult execute(
 
         var id = executeAsync(database, engine, source, readonly, inputs).transaction.id;
```

Comment:
What would you think about defining a `wait` method on `TransactionAsyncResult`, which would call `Awaitility` methods and then results, metadata, etc.? Then the body of `execute` would become:

```java
var txnAsyncRes = executeAsync(database, engine, source, readonly, inputs);
txnAsyncRes.wait();
return txnAsyncRes;
```

Reply:
That's makes sense in this case, but for the sake of simplicity and similarity with other sdks I would vote to keep it the way it is now and probably address this in a future work across all sdks

Reply:
Ok sounds good :+1: